### ISSUE TITLE
com.intellij.diagnostic.PluginException: Cannot init toolwindow net.cakebuild.toolwindow.CakeTasksWindowFactory@1ef722a4 [Plugin: net.cakebuild.cakerider]

### ISSUE THREAD

**Issue by jay-glass-cko** [cake-build/cake-rider#270]
Plugin version: `3.2.0`
IDE version: `2022.2 (RD-222.3345.139)`
Operating system: `Windows 11`
Exception: `com.intellij.diagnostic.PluginException: Cannot init toolwindow net.cakebuild.toolwindow.CakeTasksWindowFactory@1ef722a4 [Plugin: net.cakebuild.cakerider]`
Stacktrace:
```text


StackTrace too long. Please copy the StackTrace here manually.


```

**Comment by jay-glass-cko** [comment]
While trying to run a cake script (by double clicking)
- This script hasn't changed since working with a previous version of rider or the cake plugin

**Comment by nils-a** [comment]
This happens since Rider version 2022.1, when multiple plugins are installed that provide a ToolWindow with the same name.

Specifically, this error occurs, if the [CakeIntegration-Plugin](https://plugins.jetbrains.com/plugin/13237-cakeintegration) is installed alongside of Cake for Rider, as both implement a ToolWindow named 'Cake Tasks'.

Can you confirm that you have both plugins installed?

**Comment by nils-a** [comment]
Closing this as a duplicate of https://github.com/cake-build/cake-rider/issues/239. Feel free to comment if it should be re-opened.

Also, after #271 has been implemented, this error should go away and everyone who has both plugins installed will get both ToolWindows to choose from.

**Comment by jay-glass-cko** [comment]
Yup, both plugins were installed. Removing the other one resolved the issue. Thanks
